packages/console/src/components/Executions/Tables/NodeExecutionsTable.tsx

```diff
@@ -88,7 +99,7 @@ export const NodeExecutionsTable: React.FC<NodeExecutionsTableProps> = ({
   }, [initialNodes, filteredNodes, originalNodes, nodeExecutionsById]);
 
   const toggleNode = async (id: string, scopedId: string, level: number) => {
-    fetchChildrenExecutions(
+    await fetchChildrenExecutions(
```

Comment:
can you please add `await` everywhere we call `fetchChildrenExecutions`? `ExecutionTimeline` and `ReactFlowWrapper`